### PR TITLE
fix(migrations): Fixing cross filter migration

### PR DESCRIPTION
**Pull request by craig-rueda**
### SUMMARY
Fixing bug in down migration for Cross Filtering

### CODE REVIEW

**Review by betodealmeida — APPROVED** [review]
Go go go

File: superset/migrations/versions/2023-05-11_12-41_4ea966691069_cross_filter_global_scoping.py [hunk 1]
```diff
@@ -91,20 +94,24 @@ def downgrade():
...
+                if scope.lower() == "global":
                     new_chart_configuration[chart_id]["crossFilters"]["scope"] = {
                         "rootPath": ["ROOT_ID"],
                         "excluded": [chart_id],
                     }
 
             json_metadata["chart_configuration"] = new_chart_configuration
-            del json_metadata["global_chart_configuration"]
+
+            if "global_chart_configuration" in json_metadata:
```
Comment (hughhhh) [inline superset/migrations/versions/2023-05-11_12-41_4ea966691069_cross_filter_global_scoping.py:107]: nit: json_metadata.pop('global_chart_configuration')

File: superset/migrations/versions/2023-05-11_12-41_4ea966691069_cross_filter_global_scoping.py [hunk 2]
```diff
@@ -91,20 +94,24 @@ def downgrade():
...
+                del json_metadata["global_chart_configuration"]
+
```
Comment (hughhhh) [inline superset/migrations/versions/2023-05-11_12-41_4ea966691069_cross_filter_global_scoping.py:109]: nit: 
```suggestion
            json_metadata.pop('global_chart_configuration')
```

Comment (betodealmeida) [reply superset/migrations/versions/2023-05-11_12-41_4ea966691069_cross_filter_global_scoping.py:109]: `.pop()` fails if the key doesn't exist:

```python
>>> {}.pop(1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 1
```

**Comment by sadpandajoe** [comment]
🏷️ preset:2023.21

**Pull request merged**